.circleci/config.yml

```diff
@@ -0,0 +1 @@
+version: 2
```

Comment:
Pretty much every resin project runs on TravisCI (apart from something recently switching to CircleCI), let's stick with Travis for the time being, as access to it etc. is already set up & sorted for the entire team. And then we can look at switching CI services separately.